### PR TITLE
[10.0][ADD] queue_job: allow to define vacuum removal interval per channel

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
Closes https://github.com/OCA/queue/issues/187

### CODE REVIEW

**Comment by Cedric-Pigeon** [comment]
@lmignon

**Review by rousseldenis — APPROVED** [review]
Great ! Code review

**Review by lmignon — APPROVED** [review]
LGTM (Codre review only)

File: queue_job/models/queue_job.py [hunk 1]
```diff
@@ -338,6 +343,9 @@ class JobChannel(models.Model):
     job_function_ids = fields.One2many(comodel_name='queue.job.function',
                                        inverse_name='channel_id',
                                        string='Job Functions')
+    removal_interval = fields.Integer(
```
Comment (apineux) [inline queue_job/models/queue_job.py:346]: It is a personal choice but I would have chosen the name "time_to_live" instead of removal_interval.

Comment (Cedric-Pigeon) [reply queue_job/models/queue_job.py:346]: I just kept the original name

**Review by apineux — APPROVED** [review]

**Review by guewen — APPROVED** [review]

**Comment by guewen** [comment]
Great, thanks :)

/ocabot merge

**Pull request merged**